src/generators.ts

```diff
@@ -0,0 +1,48 @@
+import { generators } from 'openid-client'
+
+import * as Errors from './error'
+
+/**
+ * Generates a PKCE challenge pair where `codeChallenge` is the generated S256 challenge from `codeVerifier`
+ * @param length The length of the code verifier
+ * @returns The generated challenge pair
+ */
+export function generatePkcePair(length = 43): {
```

Comment:
thought the cool kids used arrow functions these days (either is fine)

Reply:
haha i like arrow functions, but defining function return types explicitly is recommended in library code and i feel that it looks clearer with `function` instead of arrow functions
[@typescript-eslint/explicit-module-boundary-types](https://github.com/typescript-eslint/typescript-eslint/blob/v4.25.0/packages/eslint-plugin/docs/rules/explicit-module-boundary-types.md)